Exception in thread "main" java.lang.NullPointerException
	at Influenciable.relaciona(Influenciable.java:146)
	at PruebaInfluenciableRelaciona.main(PruebaInfluenciableRelaciona.java:57)
